### PR TITLE
FINERACT-1399 Tag as non-mandatory field in Chart Of Accounts bulk up…

### PR DESCRIPTION
**Pull request by ankita10r**
…load

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by awasum** [comment]
@Cadreia , @BLasan , will you like to review this PR?

**Comment by BLasan** [comment]
> @Cadreia , @BLasan , will you like to review this PR?

Will review this today

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java [hunk 1]
```diff
@@ -231,7 +231,7 @@ private void setLayout(Sheet chartOfAccountsSheet) {
         writeString(ChartOfAcountsConstants.PARENT_COL, rowHeader, "Parent");
         writeString(ChartOfAcountsConstants.PARENT_ID_COL, rowHeader, "Parent Id");
         writeString(ChartOfAcountsConstants.ACCOUNT_NAME_COL, rowHeader, "Account Name");
-        writeString(ChartOfAcountsConstants.TAG_COL, rowHeader, "Tag *");
```
Comment (BLasan) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java:234]: Any specific reason in changing this?

**Review by ptuomola — APPROVED** [review]
LGTM

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java (hunk 1 above)
Comment (ptuomola) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java:234]: "\*" means a mandatory field, so if we are making this non-mandatory then we should remove the "\*"... 

Hope this sounds ok?

Comment (BLasan) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java:234]: Yeah, Thanks :) @ptuomola

**Review by BLasan — APPROVED** [review]

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java (hunk 1 above)
Comment (ptuomola) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/chartofaccounts/ChartOfAccountsWorkbook.java:234]: Great stuff! Will merge...

**Pull request merged**